sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -863,6 +863,21 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
          |value with same element type, but it's [array<int>, string].
        """.stripMargin.replace("\n", " ").trim()
     assert(e2.message.contains(errorMsg2))
+
+    checkAnswer(
```

Comment:
Since this is a bug, can you split these three tests into a separate test unit and add a test title with the jira ID(SPARK-29600)?

Reply:
Also, can you update the title, too?

Reply:
Sure. I'll update